pkgs/development/python-modules/wrf-python/default.nix

```diff
@@ -0,0 +1,29 @@
+{lib, fetchPypi, python, buildPythonPackage, gfortran, setuptools, mock, xarray, wrapt, numpy, fetchpatch}:
+
+buildPythonPackage rec {
+  pname = "wrf-python";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0maphsqfnrqpp52zz7xwcfjhw7y0fvq81mz91a9rfqvw0b2ww83m";
+  };
+  buildInputs = [ gfortran ];
+  propagatedBuildInputs = [
+            setuptools
+            wrapt
+            numpy>=1.11
```

Comment:
In nix you cannot specify version. We try to make everything use the latest version. `numpy>=1.11` -> `numpy`

Reply:
Ok, thanks. I realized, when testing it again this caused my build to fail. Sorry, it's my first time contributing here..